src/Magnum/EigenIntegration/Integration.h

```diff
@@ -369,6 +370,51 @@ template<class To, std::size_t size, class T> inline To cast(const Math::Vector<
     return Math::Implementation::VectorConverter<size, T, To>::to(from);
 }
 
+
+/**
+@brief Convert Corrades StridedArrayView2D to Eigens Dynamic Matrix Type.
+*/
+template<class T> inline Eigen::Map<Eigen::Matrix<T, Eigen::Dynamic, Eigen::Dynamic, Eigen::RowMajor>, Eigen::Unaligned, Eigen::Stride<Eigen::Dynamic, Eigen::Dynamic>> arrayCast(const Containers::StridedArrayView2D<T>& from) {
+    using MatrixT = Eigen::Matrix<T, Eigen::Dynamic, Eigen::Dynamic, Eigen::RowMajor>;
+    using StrideT = Eigen::Stride<Eigen::Dynamic, Eigen::Dynamic>;
+    auto size = from.size();
+    auto stride = from.stride();
```

Comment:
```suggestion
    const Containers::StridedDimensions2D<std::size_t> size = from.size();
    const Containers::StridedDimensions2D<std::ptrdiff_t> stride = from.stride();
```

I'm a `const` and `auto` extremist, sorry :P